docs/source/developer/numba-runtime.rst

```diff
@@ -184,5 +184,5 @@ The plan for NRT is to make a standalone shared library that can be linked to
 Numba compiled code, including use within the Python interpreter and without
 the Python interpreter.  To make that work, we will be doing some refactoring:
 
-* numba NPM code references statically compiled code in "helperlib.c".  Those
-  functions should be moved to NRT.
+* numba :term:`NPM` code references statically compiled code in "helperlib.c".
+Those functions should be moved to NRT.
```

Comment:
When you fixed the line wrapping here, you accidentally broke the indentation. That is why one of the tests failed. (You can also see the effect of this by watching the "Github rendered" version or the read the docs build).

I suggest you click your way through the failed test pipeline to see how the output can help with finding what's broken :-)